docs/src/configuring-zed.md

```diff
@@ -2178,6 +2178,64 @@ Float values between `0.0` and `0.9`, where:
 }
 ```
 
+## UI Font Family
+
+- Description: The name of a font to use for rendering in the UI.
+- Setting: `ui_font_family`
+- Default: `Zed Plex Sans`
```

Comment:
This isn't the default UI font anymore.

Reply:
It's not?
https://github.com/zed-industries/zed/blob/2470db490115322720f85e2ad57e922de930ee1b/assets/settings/default.json#L56

Reply:
Ah right, I was searching for "IBM" in the name.

Forgot we still had our own fork 🤦🏻‍♂️